src/Symfony/Component/Mailer/Bridge/Amazon/composer.json

```diff
@@ -20,6 +20,7 @@
         "symfony/mailer": "^4.4|^5.0"
     },
     "require-dev": {
+        "async-aws/ses": "^0.3",
```

Comment:
this is a BC break (people have to install this optional dependency) but it shouldn't be required IMO (one could use the bridge with SMTP=